doc/man1/x509.pod

```diff
@@ -354,17 +357,12 @@ can thus behave like a "mini CA".
 This option causes the input file to be self signed using the supplied
 private key.
 
-If the input file is a certificate it sets the issuer name to the
-subject name (i.e.  makes it self signed) changes the public key to the
-supplied value and changes the start and end dates. The start date is
-set to the current time and the end date is set to a value determined
-by the B<-days> option. Any certificate extensions are retained unless
-the B<-clrext> option is supplied; this includes, for example, any existing
-key identifier extensions.
-
-If the input is a certificate request then a self signed certificate
-is created using the supplied private key using the subject name in
-the request.
+It sets the issuer name to the subject name (i.e., makes it self-issued)
```

Comment:
Should the unmodified sentence above this be retained as-is?  Doesn't "self-issued" cover it?

Reply:
For some reason GitHub shows in this diff both the changes of PR#8165 and of the current PR.
What you are referring to here is changes done as part of the former PR.
What I understand you are missing here is already stated in the first sentence of the -signkey section:
> This option causes the input file to be self signed using the supplied
private key.